app/models/condition.rb

```diff
@@ -145,7 +145,7 @@ def self.collect_children(ref, methods)
     result
   end
 
-  def self._subst_find(rec, inputs, expr)
+  def self._subst_find(rec, expr)
```

Comment:
no, those methods should be declared private, like below:
 private_class_method :subst_find